src/lib/src/x86_calcbonds.cpp

```diff
@@ -5,16 +5,14 @@
 #include <cstddef>
 #include <cstdint>
 #include <immintrin.h>
+#include <iostream>
 
 #include "arrops.h"
 #include "compiler_hints.h"
 #include "distopia_type_traits.h"
-#include "ops.h"
 #include "vector_triple.h"
-#include "x86_basemath.h"
-#include "x86_swizzle.h"
-#include "x86_tgintrin.h"
-#include "x86_vector_triple_basemath.h"
+
+#include "ortho_box.h"
 
```

Comment:
Given how heavily we have templated,I think we should aim for a platform agnostic implementation of calcbonds, ie everything should (hopefully) be resolvable at a lower level than this function.